src/plane.rs

```diff
@@ -52,21 +53,41 @@ impl<T: Pixel> Debug for Plane<T>
 
 impl<T: Pixel> Plane<T> {
   /// Stride alignment in bytes.
+  #[cfg(windows)]
   const STRIDE_ALIGNMENT_LOG2: usize = 4;
+  #[cfg(not(windows))]
+  const STRIDE_ALIGNMENT_LOG2: usize = 5;
 
   /// Data alignment in bytes.
+  #[cfg(windows)]
   const DATA_ALIGNMENT_LOG2: usize = 4;
+  #[cfg(not(windows))]
+  const DATA_ALIGNMENT_LOG2: usize = 5;
```

Comment:
Is this because we aren't executing AVX2 on Windows? Not sure I'd bother with this considering we'd want to eventually have it on Windows anyway.

Reply:
This is because even without any assembly, the win64 binary segfaults with 32-byte alignment. Deferring analysis of that for when we enable nasm there.